### ISSUE TITLE
Error loading bom_rainfall_grids

### ISSUE THREAD

**Issue by Daniel-Trung-Nguyen** [GeoscienceAustralia/dea-notebooks#759]
Hi, 
I got this error when trying to load data from bom_rainfall_grids on the NCI VM. I am a member of the rr5 project
RasterioIOError: NETCDF:"/g/data/rr5/agcd/0_05/rainfall/daily/1997/rr.1997010119970101.grid.nc":rain_day: No such file or directory

Here is my code:

central_lon,central_lat = 148.2616,-22.1499
visual_buffer = 0.055
latitude = (central_lat - visual_buffer, central_lat + visual_buffer)
longitude = (central_lon - visual_buffer, central_lon + visual_buffer)
time = ('1997-01-01','2016-01-01')
query = {
    'x': longitude,
    'y': latitude,
    'time': time,
#     "group_by": 'solar_day', # Add group_by to reduce data
}   
ds2 = dc.load(product='bom_rainfall_grids', **query)

**Comment by robbibt** [comment]
@GypsyBojangles or @omad might have some thoughts on this... I suspect it's been a while since anyone checked this data

**Comment by Daniel-Trung-Nguyen** [comment]
Thanks for the reply, @robbibt! I am working on a project at QUT monitoring pasture health in relation to climate change. I would love to download the gridded maps of rainfall, temperature, humidity, and solar exposure from the BOM website (http://www.bom.gov.au/climate/maps/) to use in my analysis. Is this something that has been done for the NCI databases? If not, is there an option for me to create a datacube product for these?
Thanks,
Trung

**Comment by MatthewJA** [comment]
Are you part of rr5 on the NCI?

**Comment by Daniel-Trung-Nguyen** [comment]
Hi @MatthewJA, 
Yes, I am. However, the "agcd" folder did not show up on the g/data/rr5 in my file browser. 
![image](https://user-images.githubusercontent.com/22897462/107900447-c23fd580-6f8c-11eb-802c-97c97110bcc0.png)
![image](https://user-images.githubusercontent.com/22897462/107900506-e4d1ee80-6f8c-11eb-9b52-4fc1edaab3bb.png)

**Comment by MatthewJA** [comment]
Yep, that data seems to have been deleted. Anyone know why or where it went?

**Comment by CEKrause** [comment]
I am not sure, but I got this message from NCI the other day, that may offer an alternative? 

> Dear NCI Data Users,
> We are pleased to announce the release of the Australian Gridded Climate Data Collection.
> The Australian Gridded Climate Data (AGCD) is the Bureau of Meteorology's official dataset for climate analyses covering the variables of rainfall, temperature (maximum and minimum) as well as vapour pressure at daily and monthly timescales.
> You can register for access through the MyNCI portal.
> This collection is also available through the NCI THREDDS Data Server. Please see our Data Catalogue for further information on the data products and data access.
> For any questions, please contact help@nci.org.au.

**Comment by BexDunn** [comment]
The BoM rainfall grids on the NCI were deleted and replaced with the [Australian Gridded Climate Data (AGDC)](https://geonetwork.nci.org.au/geonetwork/srv/eng/catalog.search#/metadata/f5999_1125_5714_7440) however the replacement data is monthly and doesn't currently have daily rainfall totals.

Options for replacement in the [Estimate climate driver influence on rainfall notebook:](https://github.com/GeoscienceAustralia/dea-notebooks/blob/1899a7026cfcba478a99acee892e3e2fd79ea346/Real_world_examples/Estimate_climate_driver_influence_on_rainfall.ipynb#L10) include: [SILO](https://www.longpaddock.qld.gov.au/silo/gridded-data/),[ ECMWF Re-Analysis 5 (ERA5) ](https://geonetwork.nci.org.au/geonetwork/srv/eng/catalog.search#/metadata/f5137_3785_1482_5011), or [ANU Climate 2.0 ](https://geonetwork.nci.org.au/geonetwork/srv/eng/catalog.search#/metadata/f5027_3595_6164_9750).

Utility may depend on format and model accuracy, with potential stretch goal of introdcuing multiple options here.
